File: android/app/src/main/java/my/ssdid/mobile/domain/model/CredentialStatus.kt
package my.ssdid.mobile.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class CredentialStatus(
    val id: String,
    val type: String,
    val statusPurpose: String,
    val statusListIndex: String,
    val statusListCredential: String
)

File: sdk/ios/SsdidCore/Sources/SsdidCore/Domain/Model/CredentialStatus.swift
import Foundation

public struct CredentialStatus: Codable {
    public let id: String
    public let type: String
    public let statusPurpose: String
    public let statusListIndex: String
    public let statusListCredential: String

    public init(id: String, type: String, statusPurpose: String, statusListIndex: String, statusListCredential: String) {
        self.id = id
        self.type = type
        self.statusPurpose = statusPurpose
        self.statusListIndex = statusListIndex
        self.statusListCredential = statusListCredential
    }
}
